tests/e2e/specs/shopper/mini-cart.test.js

```diff
@@ -554,7 +558,18 @@ describe( 'Shopper → Mini Cart', () => {
 
 			await expect( page ).toMatchElement( 'h1', { text: 'Checkout' } );
 
-			await expect( page ).toMatch( productTitle );
+			const orderSummaryToggle = await page.$(
+				block.selectors.frontend.checkoutOrderSummary.toggle
```

Comment:
Was it the other PR where we destructured `selectors`? What do you think to keep using that pattern for consistency? Even though here might be less useful. Just a suggestion.

Reply:
I missed that. Thanks for spotting!